Palindrome.java

```diff
@@ -25,6 +27,64 @@
  */
 public class Palindrome {
     public static void main(String[] args) {
-        // Your code goes here :)
+
+        Scanner input  = new Scanner(System.in);
+        byte palindromeSize;
+
+        System.out.printf("Please enter a number(int), between 0 and 1,000,000,000: ");
+        long inputNum = input.nextInt();

```

Comment:
'int' is enough

Reply:
Again, was worried that a bigger number will crash the program before i have a chance to validate the input.